machines/overview.html.markerb

```diff
@@ -117,6 +117,27 @@ Updating a Machine takes it down (like with `fly machine stop`), applies configu
 not changing the image, so we don't have to go fetch it from the global registry, this is fast, for the same reason `stop` and `start`
 are; we've already done the heavy lifting.
 
+### Static Machine IPs
+
+Static egress IPs can be attached to a machine. These IPs survive a [machine migration].
```

Comment:
Does a static egress IP have any other features beyond surviving a machine migration- isn't it also unique to the machine and not shared among other customers?
(Also is `[machine migration]` supposed to link somewhere)